<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QListView" name="sectionsListView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QListView" name="notebooksListView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNotebook"/>
     <addaction name="actionSection"/>
     <addaction name="actionPage"/>
     <addaction name="separator"/>
     <addaction name="actionExplain_Choices_on_this_menu"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose_This_Notebook"/>
    <addaction name="actionNotebook_Properties"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPrint_Content_to_Log"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionNotebook_Help"/>
    <addaction name="actionKeyboard_Shortcuts"/>
    <addaction name="actionContact_Us_Contribute"/>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionAbout_Spiral"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="browserToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose_This_Notebook">
   <property name="text">
    <string>Close This Notebook</string>
   </property>
  </action>
  <action name="actionNotebook_Properties">
   <property name="text">
    <string>Notebook Properties</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNotebook">
   <property name="text">
    <string>Notebook</string>
   </property>
  </action>
  <action name="actionSection">
   <property name="text">
    <string>Section</string>
   </property>
  </action>
  <action name="actionPage">
   <property name="text">
    <string>Page</string>
   </property>
  </action>
  <action name="actionExplain_Choices_on_this_menu">
   <property name="text">
    <string>Explain Choices on this menu</string>
   </property>
  </action>
  <action name="actionPrint_Content_to_Log">
   <property name="text">
    <string>Print Content to Log</string>
   </property>
  </action>
  <action name="actionNotebook_Help">
   <property name="text">
    <string>Notebook Help</string>
   </property>
  </action>
  <action name="actionKeyboard_Shortcuts">
   <property name="text">
    <string>Keyboard Shortcuts</string>
   </property>
  </action>
  <action name="actionContact_Us_Contribute">
   <property name="text">
    <string>Contact Us / Contribute</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionAbout_Spiral">
   <property name="text">
    <string>About Spiral</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
